resources/common/labelers/oto-labeler/labeler.json

```diff
@@ -8,14 +8,16 @@
     "displayedName": {
         "en": "UTAU oto labeler",
         "zh": "UTAU oto 标注器",
-        "ja": "UTAU oto ラベラー"
```

Comment:
Could you update the version for all updated labeler/plugins?

Reply:
I've updated the version of 3 labelers and 2 plugins just now.